public/app/features/admin/state/reducers.test.ts

```diff
@@ -17,15 +17,15 @@ import { LdapState, LdapUser, UserAdminState, UserDTO, UserListAdminState } from
 
 const makeInitialLdapState = (): LdapState => ({
   connectionInfo: [],
-  syncInfo: null,
-  user: null,
-  ldapError: null,
-  connectionError: null,
-  userError: null,
+  syncInfo: undefined,
+  user: undefined,
+  ldapError: undefined,
+  connectionError: undefined,
+  userError: undefined,
```

Comment:
Do we actually need these props to be declared on default state if they're undefined?